src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.color.ColorSuplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    
+    public Figure getRandomFigure() {
+        return getFigure();
+    }
+
+    private Figure getFigure() {
+        final int figures = 5;
+        final int maxSize = 99;
+        ColorSuplier colorSuplier = new ColorSuplier();
```

Comment:
class level private final field, same as Random

Reply:
not fixed

Reply:
not fixed

Reply:
please move colorSupplier to the class fields. Also please do not make RANDOM a constant. It should be just a regular field